### PR TITLE
Add resource group to web ui

### PR DESCRIPTION
**Pull request by nezihyigitbasi**
Fixes part of #5194

### CODE REVIEW

File: presto-main/src/main/resources/webapp/assets/query.js [hunk 1]
```diff
@@ -1015,6 +1015,18 @@ var QueryDetail = React.createClass({
                                         { query.session.source }
                                     </td>
                                 </tr>
+                                  {(() => {
```
Comment (raghavsethi) [inline presto-main/src/main/resources/webapp/assets/query.js]: I don't think you need the `if`. Allowing the field to be null is fine, and more consistent with the rest of the UI.

**Comment by raghavsethi** [comment]
Does this also fix #7049? If not, maybe that's worth doing alongside this?

**Comment by nezihyigitbasi** [comment]
@raghavsethi Nope this doesn't fix #7049. I have another PR (#6483) thats add a REST endpoint for that.

**Comment by nezihyigitbasi** [comment]
@raghavsethi Addressed your comment & rebased.

File: presto-main/src/main/resources/webapp/assets/query.js (hunk 1 above)
Comment (raghavsethi) [inline presto-main/src/main/resources/webapp/assets/query.js]: Do you still need the lambda?

**Comment by nezihyigitbasi** [comment]
@raghavsethi lambda is not needed you are right, so removed it.

**Comment by raghavsethi** [comment]
Merged, thanks!

**Pull request closed without merging**